### PR TITLE
Feat exclude path

### PR DESCRIPTION
**Pull request by xavierchow**

### CODE REVIEW

File: test/app.e2e-spec.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+describe('AppController (e2e)', () => {
+  let app: INestApplication;
+  beforeAll(async () => {
+    const moduleFixture: TestingModule = await Test.createTestingModule({
+      imports: [AppModule],
+    }).compile();
+    app = moduleFixture.createNestApplication();
+    await app.init();
+  });
+
```
Comment (jiangmengyu0409) [inline test/app.e2e-spec.ts:16]: Usually there will be a `afterAll` which will call the `await app.close()` method to close the app.

**Review by jiangmengyu0409 — APPROVED** [review]

File: test/app.e2e-spec.ts (hunk 1 above)
Comment (xavierchow) [reply test/app.e2e-spec.ts:16]: sure, adding

File: src/logger.interfaces.ts [hunk 2]
```diff
@@ -12,6 +12,7 @@ export interface LoggerConfig {
   excludeHeaders?: string[];
   reqIdHeader?: string;
   genReqId?: (r: Request) => string;
+  excludeReqPath?: string;
```
Comment (bsdelf) [inline src/logger.interfaces.ts:15]: I suggest use array for extensibility

```
  excludeReqPaths?: string[];
```

Comment (xavierchow) [reply src/logger.interfaces.ts:15]: I was thinking about a regexp as an enhancement, but I will leave it until the real case appears.
At the moment, the health check API is the only case we need to skip the logging.

**Review by bsdelf — APPROVED** [review]

**Pull request merged**